From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Alexandre Derumier <aderumier@odiso.com>
Date: Wed, 16 May 2018 02:16:42 +0200
Subject: [PATCH] netlink : IFLA_BRPORT_ARP_SUPPRESS : use 32

this has been upstreamed recently in linux kernel, with IFLA_BRPORT_NEIGH_SUPPRESS.
cumulus was using a temp 152 number.
It's need for vxlan + anycast gateway
---
 ifupdown2/nlmanager/nlpacket.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ifupdown2/nlmanager/nlpacket.py b/ifupdown2/nlmanager/nlpacket.py
index d6f93fe..28c43c2 100644
--- a/ifupdown2/nlmanager/nlpacket.py
+++ b/ifupdown2/nlmanager/nlpacket.py
@@ -3306,9 +3306,9 @@ class Link(NetlinkPacket):
     IFLA_BRPORT_VLAN_TUNNEL         = 29
     IFLA_BRPORT_BCAST_FLOOD         = 30
     IFLA_BRPORT_GROUP_FWD_MASK      = 31
+    IFLA_BRPORT_ARP_SUPPRESS        = 32
     IFLA_BRPORT_PEER_LINK           = 150
     IFLA_BRPORT_DUAL_LINK           = 151
-    IFLA_BRPORT_ARP_SUPPRESS        = 152
     IFLA_BRPORT_GROUP_FWD_MASKHI    = 153
     IFLA_BRPORT_DOWN_PEERLINK_REDIRECT = 154
 
-- 
2.11.0

